projects/aca-content/src/lib/components/search/search-input-control/search-input-control.component.spec.ts

```diff
@@ -42,14 +42,24 @@ describe('SearchInputControlComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should emit submit event on searchSubmit', () => {
-    component.searchTerm = 'mock-search-term';
+  it('should emit submit event if form is valid', () => {
+    component.searchTerm = 'valid';
+
+    let submittedTerm = '';
+    component.submit.subscribe((term) => (submittedTerm = term));
```

Comment:
instead subscribe - can you please spyOn emit function on submit and check using expect if emit function is called with proper parameter?